[15.0][ADD] sale_order_line_menu: New views and stored fields (#2591)
OCA/sale-workflow: sale_order_line_menu/readme/DESCRIPTION.rst

> Adds a menu item and a couple of views to navigate through Sale Order lines.

Reviewer: Just in case tomorrow they are 3... 😋 

```suggestion
Adds a menu item and some views to navigate through Sale Order lines.
```
